### PR TITLE
task 7 done with tests

### PR DESCRIPTION
**Pull request by samelawton**

### CODE REVIEW

File: __tests__/app.test.js [hunk 1]
```diff
@@ -179,4 +179,46 @@ describe('app',() => {
         })
         
     })
+    describe.only('POST- /api/reviews/:review_id/comments', ()=>{
```
Comment (yalicee) [inline __tests__/app.test.js]: remove .only

File: __tests__/app.test.js [hunk 2]
```diff
@@ -179,4 +179,46 @@ describe('app',() => {
...
+            .expect(201)
+            .then(({body})=>{
+                const comment = body.commentInfo;
+                console.log(comment)
+                expect(comment).toHaveLength(1);
+                expect(comment[0]).toMatchObject({
+                    comment_id: expect.any(Number),
+                    body: expect.any(String),
+                    votes: expect.any(Number),
+                    author : expect.any(String),
```
Comment (yalicee) [inline __tests__/app.test.js:200]: You know the author and the body so test exactly for that. Also remove the console.log

**Review by yalicee — CHANGES_REQUESTED** [review]
You need the following tests:
Status 400, missing required field(s), e.g. no username or body properties
Status 404, username does not exist

**Review by johnhutcheon — COMMENTED** [review]
Hey Sam, thanks for making the changes, good to merge 👍

**Review by johnhutcheon — APPROVED** [review]

**Pull request merged**